src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubExtensionConfigProvider.cs

```diff
@@ -72,6 +73,9 @@ public void Initialize(ExtensionConfigContext context)
             context.AddBindingRule<EventHubAttribute>()
                 .BindToCollector(BuildFromAttribute);
 
+            // register EventClient output binding
```

Comment:
nit: I'd just remove this comment. It's not technically an output binding, it's a client/service binding